binary-search-trees/03-Binary-Search-Trees-Traversal.md

```diff
@@ -293,4 +627,37 @@ With breadth first search, we need to consider the maximum number of nodes that
 
 ## Summary
 
-In the lesson, we examined different methods to traverse a tree.  Unlike a linked list where there is only one possible traversal method, a tree has multiple ways to traverse.
+In the lesson, we examined different methods to traverse a tree.  Unlike a linked list where there is only one possible traversal method, a tree has multiple ways to traverse. The type of traversal we use depends on the task we are trying to complete. 
+
+### !challenge
+
+* type: paragraph
+* id: 34de5a16-365c-47e5-bba1-e7970c65623e
+* title: Binary Search Trees Reflection
+* points: 0
+<!-- * topics: [python, pandas] (Checkpoints only, optional the topics for analyzing points) -->
+
+##### !question
+
+Take 5 minutes to review the above lesson and write down any questions you still have about the material. Is there anything that needs more clarification or you would like to go over again?
+
+Bring these questions to class! If reviewing this material after class, bring these questions to the #study-hall Slack channel or ask in office hours.
+
+##### !end-question
+
+##### !placeholder
+
+Ex. I'm still not clear on how the call stack works with DFS traversals.  
+<br>
```

Comment:
This <br> tag showed up in the textbox. You might be able to remove it and maintain formatting by adding a few extra spaces after the first line.